examples/simple/src/dataProvider.tsx

```diff
@@ -57,7 +57,7 @@ const addTagsSearchSupport = (dataProvider: DataProvider) => ({
                     ...params,
                     filter: item => {
                         const matchPublished =
-                            item.published == params.filter.published; // eslint-disable-line eqeqeq
```

Comment:
I'm not sure about this: we got issues because of using `==` instead of `===`. Shouldn't we add this rule in the config?